go.mod

```diff
@@ -5,7 +5,7 @@ go 1.16
 require (
 	cloud.google.com/go v0.71.0 // indirect
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.10.1
-	github.com/heimweh/go-pagerduty v0.0.0-20220422231448-43095fe5ba3f
+	github.com/heimweh/go-pagerduty v0.0.0-20220428180718-5a69bb821163
```

Comment:
This version needs to be updated after [we merge ](https://github.com/heimweh/go-pagerduty/pull/88) is merged